src/Microsoft.DotNet.XHarness.iOS.Shared/Logging/ILogs.cs

```diff
@@ -8,22 +8,22 @@
 
 namespace Microsoft.DotNet.XHarness.iOS.Shared.Logging
 {
-    public interface ILogs : IList<ILog>, IDisposable
+    public interface ILogs : IList<IFileBackedLog>, IDisposable
```

Comment:
I guess these are then `IFileBackedLogs`?

Reply:
Also wondering - why not just `IFileLog`?

Reply:
I tried, less freedom :/